### PR TITLE
Fix sync connect for small screens

### PR DESCRIPTION
**Pull request by karlenDimla**
Task/Issue URL: https://app.asana.com/0/1201493110486074/1206542898886222/f

### Description
Fix layout for sync connect screen in smaller devices

### Steps to test this PR
- [ ] Go to Settings
- [ ] Open Sync & Backup
- [ ] Click on Sync with Another Device
- [ ] Verify that UI is correct

Results: https://app.asana.com/0/0/1206948358341237/f

### CODE REVIEW

**Comment by karlenDimla** [comment]
* **#4365** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/4365?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @karlenDimla and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: sync/sync-impl/src/main/res/layout/activity_connect_sync.xml [hunk 1]
```diff
@@ -1,5 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!--
+<?xml version="1.0" encoding="utf-8"?><!--
```
Comment (malmstein) [inline sync/sync-impl/src/main/res/layout/activity_connect_sync.xml:1]: what’s with the format change?

**Review by malmstein — APPROVED** [review]
works as expected. small format comment

File: sync/sync-impl/src/main/res/layout/activity_connect_sync.xml (hunk 1 above)
Comment (karlenDimla) [reply sync/sync-impl/src/main/res/layout/activity_connect_sync.xml:1]: Not sure. Probably just automatically changed when I formatted the file. @malmstein  should I revert it?

**Pull request merged**